docs/examples/jaeger-docker.md

```diff
@@ -0,0 +1,117 @@
+# Collecting traces using Jaeger
+
+In this example, we are going to use [Jaeger](https://jaegertracing.io/) to collect and visualize traces produced by an
+application.
+We will cover the installation and configuration of Jaeger, as well as the instrumentation of the application using the
+otel4s library.
+
+### Project setup
+
+Configure the project in the `build.sbt`:
+
+```scala
+libraryDependencies ++= Seq(
+  "org.typelevel" %% "otel4s-java-auto" % "@VERSION@", // <1>
+  "io.opentelemetry" % "opentelemetry-exporter-otlp" % "@OPEN_TELEMETRY_VERSION@" // <2>
+)
+run / fork := true
+javaOptions += "-Dotel.java.global-autoconfigure.enabled=true" // <3>
+javaOptions += "-Dotel.service.name=jaeger-example"            // <4>
+javaOptions += "-Dotel.metrics.exporter=none"                  // <5>
+```
+
+1) Add `otel4s` library  
+2) Add an OpenTelemetry exporter. Without the exporter, the application will crash  
+3) Enable OpenTelemetry SDK autoconfigure mode  
+4) The name of the application to use in the traces  
+5) Disable metrics exporter since Jaeger is compatible only with traces  
+
+### OpenTelemetry SDK configuration
+
+As mentioned above, we use `otel.service.name` and `otel.metrics.exporter` system properties to configure the
+OpenTelemetry SDK.
+The SDK can be configured via environment variables too. Check the full list
+of [environment variable configurations](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md)
+for more options.
+
+### Jaeger configuration
+
+In order to collect and visualize traces, you can run [Jaeger](https://jaegertracing.io/)
+using [Docker](https://www.docker.com/).
+
+```shell
+$ docker run --name jaeger \
+  -e COLLECTOR_OTLP_ENABLED=true \
+  -p 16686:16686 \
+  -p 4317:4317 \
+  -p 4318:4318 \
+  jaegertracing/all-in-one:1.35
+```
+
+1) `-e COLLECTOR_OTLP_ENABLED=true` - enable OpenTelemetry receiver  
```

Comment:
Did we want to link this list up to the above with shell comments like we did with the build file up above?  For example # <1> and so on?

Reply:
I wanted initially. Unfortunately, you can not copy-paste the example right to the shell. Neither bash nor zsh does treat `#` as a comment.

```
docker run --name jaeger \
  -e COLLECTOR_OTLP_ENABLED=true \ # <1>
  -p 16686:16686 \ # <2>
  -p 4317:4317 \ # <3>
  -p 4318:4318 \ # <3>
  jaegertracing/all-in-one:1.35
  ```
  
 bash:
 ```
 bash-5.1$ docker run --name jaeger \
  -e COLLECTOR_OTLP_ENABLED=true \ # <1>
  -p 16686:16686 \ # <2>
  -p 4317:4317 \ # <3>
  -p 4318:4318 \ # <3>
  jaegertracing/all-in-one:1.35
bash: syntax error near unexpected token `1'
bash: syntax error near unexpected token `2'
bash: syntax error near unexpected token `3'
bash: syntax error near unexpected token `3'
```

Reply:
Good enough reason to leave it out, thanks for the explanation!